README.md

```diff
@@ -195,9 +195,9 @@ newtype TeletypeIOC m a = TeletypeIOC { runTeletypeIOC :: m a }
 instance (Carrier sig m, MonadIO m) => Carrier (Teletype :+: sig) (TeletypeIOC m) where
   ret = TeletypeIOC . ret
 
-  eff = TeletypeIOC . (alg \/ eff . handleCoercible)
-    where alg (Read    k) = liftIO getLine      >>= runTeletypeIOC . k
-          alg (Write s k) = liftIO (putStrLn s) >>  runTeletypeIOC   k
+  eff = TeletypeIOC . handleSum (eff . handleCoercible) (\ t -> case t of
```

Comment:
I opted not to use `LambdaCase` in the README to avoid confusing readers unfamiliar with the extension.